### PR TITLE
Implement abstract process method for BatchHandlerTrait

### PR DESCRIPTION
**Pull request by babeuloula**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | /
| License       | MIT
| Doc PR        | /

Hello,

Now with PHP 8.0, traits support the use of abstract methods for private and protected. Just a simple improvement.

Have a nice day.

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

**Comment by SHTIKOV** [comment]
I think that it is not worth giving up support for previous versions, since not everyone uses version 8.x and more

**Comment by babeuloula** [comment]
> I think that it is not worth giving up support for previous versions, since not everyone uses version 8.x and more

Hello @SHTIKOV, Symfony 6 works only with > PHP 8.

**Comment by SHTIKOV** [comment]
> > I think that it is not worth giving up support for previous versions, since not everyone uses version 8.x and more
> 
> Hello @SHTIKOV, Symfony 6 works only with > PHP 8.

sorry i didn't notice the version

**Review by welcoMattic — APPROVED** [review]

**Comment by welcoMattic** [comment]
Thank you @babeuloula.

File: src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php [hunk 1]
```diff
@@ -11,13 +11,18 @@
...
  * @author Nicolas Grekas <p@tchwork.com>
  */
 trait BatchHandlerTrait
 {
+    /**
+     * Completes the jobs in the list.
+     *
+     * @list<array{0: object, 1: Acknowledger}> $jobs A list of pairs of messages and their corresponding acknowledgers
+     */
+    abstract private function process(array $jobs): void;
```
Comment (lyrixx) [inline src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php:24]: methods go after properties, aren't?

Comment (babeuloula) [reply src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php:24]: Not with fabbot.io

Comment (fabpot) [reply src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php:24]: They should yes

Comment (fabpot) [reply src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php:24]: And we never reorganize the code as it makes diff more difficult to read and merge more complex

Comment (babeuloula) [reply src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php:24]: I can create a new pull request with the correct order if you want.

Comment (lyrixx) [reply src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php:24]: yes, please :)

Comment (babeuloula) [reply src/Symfony/Component/Messenger/Handler/BatchHandlerTrait.php:24]: The new PR : https://github.com/symfony/symfony/pull/44287

**Pull request merged**